### ISSUE TITLE
Chrome Web - Copy/pasting brings up a unclear image past growl error

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#11131]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open up any chat in Chrome
2. Drag-select any text in a chat
3. Copy with Command-C
4. Paste into the compose box with Command-V
5. Note the weird error message in the upper right corner

## Expected Result:
Massage should paste without any issue 

## Actual Result:
Error displayed when user tries to paste 

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 1.2.1-0
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
![image - 2022-09-20T012325 520](https://user-images.githubusercontent.com/43995119/191174296-b84b5b6b-abe3-48c1-8b1d-1990e9e7967f.png)
![image - 2022-09-20T012324 205](https://user-images.githubusercontent.com/43995119/191174306-c3488792-b7bb-409e-802f-7169472d1f60.png)

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @quinthar 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1663278146401119

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by amyevans** [comment]
I've tried reproducing locally, on staging, and on production with a number of different test strings, including the exact string from the screenshot (`Testing 2 3 4 (sending while dbarrett@expensify.com app is not running)`) but haven't been able to reproduce.

The growl error is triggered here: https://github.com/Expensify/App/blob/main/src/components/Composer/index.js#L306, so following that code, the DOM parser found images in it - but obviously looking at the string there are no visible images.

@quinthar can you provide any additional details that might help in reproducing this?

**Comment by amyevans** [comment]
Additionally, @mvtglobally, was the QA team able to reproduce this?

**Comment by amyevans** [comment]
Also reported by @coleaeason [in Slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1663708415670229). Same question - any additional details you can provide to help in making this reproducible?

**Comment by amyevans** [comment]
I've also now tested sending the text from an Android simulator in case that's the source of the issue, but still unable to reproduce. Logging [this line](https://github.com/Expensify/App/blob/main/src/components/Composer/index.js#L284), we get:
```
<meta charset='utf-8'><comment><span>Testing 2 3 4 (sending while </span><div><a href="mailto:dbarrett@expensify.com">dbarrett@expensify.com</a></div><span> app is not running)</span><br></comment>
```

**Comment by mvtglobally** [comment]
@amyevans i was able to reproduce at the time of logging this issue. I am not able to repro now, but I will also ask team to check if anyone can reproduce

**Comment by amyevans** [comment]
Sounds good, thank you @mvtglobally!

**Comment by amyevans** [comment]
Demoting to weekly until we have proper repro confirmed

**Comment by mvtglobally** [comment]
@amyevans 
It doesn't seem to be repro any more 

https://user-images.githubusercontent.com/43995119/194129905-5d3dffc9-3d3d-4e12-9002-632a4f588c05.mp4

**Comment by coleaeason** [comment]
I think these are the reproduction steps:

1. Have a message thread where one person has sent 2 messages
2. the opposite person responds
3. Click to highlight the *bottom* message from person 1
4. Paste the message in the compose box, get growl.

**Comment by amyevans** [comment]
Got it, reproduced with those steps! Thanks @coleaeason

![Screen Shot 2022-10-05 at 4 49 04 PM](https://user-images.githubusercontent.com/7277067/194160757-e88ac09f-d09e-4960-98e6-d078d1061f4f.png)

I'll update the OP and release for external proposals.

**Comment by abekkala** [comment]
Reassigning the `External` label as this is a Daily and I'm ooo

**Comment by abekkala** [comment]
Oh, I think @NikkiWines is already considered the `External` assignment anyway??

**Comment by thesahindia** [comment]
Not overdue. No proposal yet.

**Comment by NikkiWines** [comment]
Dropping to weekly while we wait for proposals.

**Comment by NikkiWines** [comment]
Yep! That looks good! Re-adding the `External` label to get a CM on this

**Comment by NikkiWines** [comment]
@isabelastisser could you assign @mollfpr to this job, please? Thank you!

**Comment by abekkala** [comment]
@NikkiWines do you think when this fix is out it will also resolve the one I've noted above?

**Comment by thesahindia** [comment]
Hey @abekkala, that's a different issue and won't be fixed by it.

**Comment by isabelastisser** [comment]
Reviewing this now.

**Comment by isabelastisser** [comment]
Internal job posting: https://www.upwork.com/ab/applicants/1582893872918482944/job-details

External job posting: https://www.upwork.com/jobs/~014d8d5232f2ca4ab3

**Comment by mollfpr** [comment]
@isabelastisser Applied, thank you!

**Comment by isabelastisser** [comment]
Hey @mollfpr:

![2022-10-21_06-26-35](https://user-images.githubusercontent.com/31258516/197174901-c7bd2af0-ce85-4967-b760-401fd1156ca0.png)

**Comment by isabelastisser** [comment]
Update: I hired @mollfpr to fix the issue and @thesahindia as Contributor +. 

@mvtglobally , I can't seem to find you on Upwork, can you please apply for the job so that I can hire you? Thanks!

**Comment by isabelastisser** [comment]
Sorry for the delay, payment will be issued today.

**Comment by isabelastisser** [comment]
Update:@mollfpr and @thesahindia have been paid in Upwork.

@mvtglobally, I still can't find you on Upwork. Can you please apply for the job so that I can hire you? Thanks!

CC @NikkiWines @amyevans, do you have access to Upwork? Are you able to find @mvtglobally there?

**Comment by amyevans** [comment]
@isabelastisser, @mvtglobally is a member of the Applause QA team 😄 (so no payment via Upwork)

**Comment by isabelastisser** [comment]
Oh ok! I'm still learning the process, thanks Amy! :)

**Comment by isabelastisser** [comment]
The contract has been completed, and the Upwork post has been closed.
